src/components/TabsContent/TabsContent.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TabsContent = ({ tabs, currentTabId }) => (
+  <>
+    <p>{tabs.find(tab => tab.id === currentTabId).content}</p>
```

Comment:
better to pass content to this component
TabsContent shouldn't receive all tabs